Drivers/Src/Display.c

```diff
@@ -114,13 +118,20 @@ DisplayError_t Display_Fault(os_error_loc_t osErrCode, fault_bitmap_t faultCode)
 	return DISPLAY_ERR_NONE;
 }
 
-void assertDisplayError(DisplayError_t err){
-	OS_ERR os_err;
+static void callback_displayError(void){
```

Comment:
Both `callback_displayError` and `callback_updateDisplayError` seem to be doing the same thing. Are both needed?

Reply:
Yeah, they're the exact same, but we have two of them because they're each restricted to separate files. Should we try to somehow let the callback and assert functions be shared between display-related files?

Reply:
Whoops never mind, you're right- they do the same thing, so I changed them to only use callback_displayError